modules/sgct/readme.md

```diff
@@ -0,0 +1,3 @@
+# sgct Module
+
+Description of the sgct module
```

Comment:
Update. Would be good with an explanation of what is needed to run in a cluster environment